tidb/README.md

```diff
@@ -0,0 +1,180 @@
+# Agent Check: TiDB
+
+## Overview
+
+TiDB check monitors the overall health and performance of a [TiDB][1] cluster.
+
+## Setup
+
+### Installation
+
+Since TiDB check has not been added to [Datadog Agent][8] yet, users need to manually install TiDB check.
+
+1. Install the [developer toolkit](https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
+ on any machine.
+
+2. Run `ddev release build tidb` to build the package.
+
+3. [Download the Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+
+4. Upload the build artifact to any host with an Agent and run `datadog-agent integration install -w path/to/tidb/dist/<ARTIFACT_NAME>.whl`.
```

Comment:
No need to build it manually, since the wheel will be built after the PR is merged. Then users will be able to install it like:
`datadog-agent integration install -t datadog-tidb==<INTEGRATION_VERSION>`